docs/book/cookbook/injecting-data-into-templates.md

```diff
@@ -0,0 +1,6 @@
+<noscript><meta http-equiv="refresh" content="0; url=/zend-expressive/v3/cookbook/access-common-data-in-templates/"></noscript>
```

Comment:
This file is not needed. We only have the existing redirect files as we added versioning at a later date. (The redirects are from the non-versioned pages to the versioned ones.)